src/Eloquent/ImageCollection.php

```diff
@@ -123,6 +121,13 @@ public function purgeRemovedImages()
         }
     }
 
+    public function remove()
```

Comment:
should we be guarding in code against trying to remove a fallback image or is that a non-issue (if we tried to remove one with read only creds it would be an uncaught exception?)

Reply:
non-issue, remove operations will try to remove images from the primary (local) filesystem, "fallback" support (or even knowledge of there being an alternative filesystem), is something that is only present in the Controller (ie: read operations)